Interface.C

```diff
@@ -140,12 +169,8 @@ void preciceAdapter::Interface::configureMesh(const fvMesh& mesh)
             // Assign the (x,y,z) locations to the vertices
             // TODO: Ensure consistent order when writing/reading
             for (int i = 0; i < faceNodes.size(); i++)
-            {
-                vertices[verticesIndex++] = faceNodes[i].x();
-                vertices[verticesIndex++] = faceNodes[i].y();
-                if (dim_ == 3)//out-of-plane axis = z!
-                    vertices[verticesIndex++] = faceNodes[i].z();
-            }
+                for (unsigned int d = 0; d < dim_; ++d)
+                  vertices[verticesIndex++] = faceNodes[i][d];
```

Comment:
Two questions:
1. Is the `[i][d]` form safe? Is it used elsewhere in OpenFOAM? I am mostly worried about accidentally changing some type in the future.
2. Since we restrict anyway the user to XY, what is the motivation for changing this?

Reply:
I have not made any safety check. Regarding point 2: We compress 6 locs to 2 locs, it has less duplication and is easier to read.

Reply:
I see the "less duplication" point, I am not so sure about the "easier to read", as in the current code we have explicit calls to `.x()`, `.y()`, `.z()`.

I guess it should be safe enough, let's use it. But then I guess we use `.x()` etc also in other parts of the adapter.

Reply:
It's anyway mixed, at the moment, as I use the same code already for the FSI module :sweat_smile: